src/main/resources/localstack/init-aws.sh

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env bash
 
 aws s3 --endpoint-url http://localhost:5001 mb s3://test-core-bucket
-aws s3 --endpoint-url http://localhost:5001 cp /s3/core/ s3://test-core-bucket/ --recursive
+aws s3 --endpoint-url http://localhost:5001 cp /s3/core/core/ s3://test-core-bucket/ --recursive
```

Comment:
Is this right?

Reply:
It was needed to get both uid2-admin and uid2-optout's S3 initialisation scripts coexist, but I've realised I didn't have to make this change, so I'll be reverting this